openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelDetail.component.tsx

```diff
@@ -379,32 +383,51 @@ const MlModelDetail: FC<MlModelDetailProp> = ({
   const getMlHyperParametersColumn: ColumnsType<MlHyperParameter> = useMemo(
     () => [
       {
-        title: 'Name',
+        title: t('label.name'),
         dataIndex: 'name',
         key: 'name',
       },
       {
-        title: 'Value',
+        title: t('label.value'),
         dataIndex: 'value',
         key: 'value',
       },
     ],
     []
   );
 
+  const getEmptyPlaceholder = useMemo(
+    () => () => {
+      return (
+        <ErrorPlaceHolder size={SIZE.MEDIUM}>
+          <Typography.Paragraph>
+            {t('label.no-data-available')}{' '}
+          </Typography.Paragraph>
+        </ErrorPlaceHolder>
+      );
+    },
+    []
+  );
```

Comment:
@Ashish8689 If you want to memoise a function then it's advised to use `useCallback`
And would advise to separate above code to util so that it just execute once only instead re-rendering

Reply:
@chirag-madlani  cannot use this in utils as the function contain react hook so, have used them independently in the code.

Reply:
@Ashish8689 for translation you can do something like below:

```
import i18n from "i18next";

i18n.t("label.no-data-available")

```

Reply:
@chirag-madlani  updated